modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -268,6 +268,8 @@ public void processOpts() {
         }
         if (additionalProperties.containsKey(USE_ES6)) {
             setUseES6(convertPropertyToBooleanAndWriteBack(USE_ES6));
+        } else {
+            setUseES6(false);
```

Comment:
Can the `useES6` initialization value be set to `false` to avoid this explicit call? It just seems to be easier to reason about the code if there is some default which changes only when the config option is present.